### PR TITLE
Update File 1

### PR DESCRIPTION
**Pull request by Bharathdevarajan**
this is my first pull request

### CODE REVIEW

File: File 1 [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Emoloyee Name :
-Employee ID : 
+Emoloyee Name : Bharath D
```
Comment (ImthiyasMansur20) [inline File 1:1]: add two more variables

Comment (Bharathdevarajan) [reply File 1:1]: i have Added 2 variables

Comment (ImthiyasMansur20) [reply File 1:1]: Okay

**Review by Bharathdevarajan — COMMENTED** [review]
I have done all the changes. you can approve now.

**Comment by ImthiyasMansur20** [comment]
sure i will merge

**Comment by ImthiyasMansur20** [comment]
i will start a review

**Comment by ImthiyasMansur20** [comment]
send me a details of employees

**Pull request merged**